torch/_inductor/decomposition.py

```diff
@@ -71,6 +71,7 @@
 decomps_to_exclude = [
     aten._unsafe_index,
     aten._scaled_dot_product_flash_attention.default,  # See comments in torch/_decomp/decompositions.py
+    aten._unsafe_view,
```

Comment:
Why are you not decomposing it in inductor?

Reply:
Inductor has a lowering for this:

https://www.internalfb.com/code/fbsource/[ba02efe6ad091b414cb786eaf9d6172fe887b857]/xplat/caffe2/torch/_inductor/lowering.py

According to Sherlock, 

> Inductor has lowering for this: implying inductor would want this op preserved in graph.
> Please exclude this from inductor decomp.

https://github.com/pytorch/pytorch/pull/108546?fbclid=IwAR0PhDU5AAJRiFPRcIQyU9r-rPd2ARaF2T41xMYv-YNTtkWfRIjdm98L3xo#discussion_r1320206370

Reply:
This makes pattern matching more difficult, we should be including it.